### PR TITLE
Update github-token-available-permissions.md

### PR DESCRIPTION
**Pull request by ghost**
### Why:

Closes [issue link]

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [x] This pull request impacts the contribution experience
  - [x] I have added the 'writer impact' label
  - [x] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by PressureTechs** [comment]
fix

File: data/reusables/github-actions/github-token-available-permissions.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ permissions:
   checks: read|write|none
   contents: read|write|none
   deployments: read|write|none
+  discussions: read|write|none
```
Comment (meshl555) [inline data/reusables/github-actions/github-token-available-permissions.md:9]: Thanks

**Review by meshl555 — COMMENTED** [review]
None.

**Review by meshl555 — COMMENTED** [review]
None.

**Comment by kissana37** [comment]
[Android](Mistore.com)

**Pull request closed without merging**